### PR TITLE
Apply visual suggestions on /pro page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Apply visual suggestions on /pro page according to [Figma](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=3227%3A17378&t=7t0zBSmVPUroFrDB-0) and [mobile view](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=3299%3A17383&t=4gHFFXCn1FJAV9PI-0)

## QA

- Go to `/pro` page or https://ubuntu-com-12494.demos.haus/pro
- Compare page against design
- FYI, in Figma `Ready for development` is the design we are aiming for and `QA` is things that needs to be fixed.

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12473 and https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-1805

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@minkyngkm anything blocking this?

**Review by abhigyanghosh30 — APPROVED** [review]

File: static/sass/styles.scss [hunk 1]
```diff
@@ -400,6 +400,48 @@ summary {
...
+    padding: calc(#{$spv--large} - 1px) calc(#{$spv--large} - 1px)
+      $spv--strip-deep * 0.5;
+    @media (min-width: $breakpoint-small) {
+      padding: calc(#{$spv--small} - 1px) calc(#{$spv--large} - 1px)
+        $spv--strip-deep;
+    }
+  }
+}
+.bg-light {
+  background-color: #f7f7f7;
```
Comment (lyubomir-popov) [inline static/sass/styles.scss:415]: this should use the correct variable from vanilla

File: static/sass/styles.scss [hunk 2]
```diff
@@ -400,6 +400,48 @@ summary {
...
+}
+
+.bg-mid-dark {
+  background-color: #666;
```
Comment (lyubomir-popov) [inline static/sass/styles.scss:419]: please use the correct variable

File: static/sass/styles.scss [hunk 3]
```diff
@@ -400,6 +400,48 @@ summary {
...
+  color: white;
+  hr {
+    background-color: rgba(217, 217, 217, 0.2);
```
Comment (lyubomir-popov) [inline static/sass/styles.scss:422]: again, pleaase use the corrct variable 
```suggestion
    background-color: $colors--dark-theme--border-low-contrast;
```

File: static/sass/styles.scss (hunk 2 above)
Comment (lyubomir-popov) [inline static/sass/styles.scss:419]:
```suggestion
  background-color: $color-mid-dark;
```

File: static/sass/styles.scss [hunk 4]
```diff
@@ -400,6 +400,48 @@ summary {
...
+  }
+}
+
+.bg-dark {
+  background-color: black;
+  color: white;
+  hr {
+    background-color: rgba(217, 217, 217, 0.2);
```
Comment (lyubomir-popov) [inline static/sass/styles.scss:430]:
```suggestion
    background-color: $colors--dark-theme--border-low-contrast;
```

File: static/sass/styles.scss (hunk 1 above)
Comment (lyubomir-popov) [inline static/sass/styles.scss:415]:
```suggestion
  background-color: $color-light;
```

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]
Some suggestions in the code

**Comment by lyubomir-popov** [comment]
The diff here is quite large, has this been rebased?

**Comment by minkyngkm** [comment]
Thanks @abhigyanghosh30 and @lyubomir-popov for reviewing! Everything mentioned has been addressed. Could you take a final look please @lyubomir-popov before merging? I've also merged the latest commit from main and resolved conflicts. Should be all good now 👍

**Comment by lyubomir-popov** [comment]
LGTM, +1

**Pull request merged**